pkg/commands/workload_test.go

```diff
@@ -1674,7 +1674,12 @@ Published source
 
 			cmd := &cobra.Command{}
 			ctx := cli.WithCommand(context.Background(), cmd)
-			ctx = source.StashGgcrRemoteOptions(ctx, remote.WithTransport(registry.Client().Transport))
+			ctx = source.StashContainerRemoteTransport(ctx, reg.Client().Transport)
+			ctx = source.StashRegistryOptions(ctx, registry.Opts{
+				VerifyCerts:           true,
+				RetryCount:            2,
```

Comment:
@odinnordico : I changed the defaults to what is in imgpkg CLI flags. Let me know if this is ok